dds/DdsDynamicDataSeq.idl

```diff
@@ -42,12 +42,12 @@ module DDS {
   typedef sequence<double> Float64Seq;
   typedef sequence<long double> Float128Seq;
   typedef sequence<char> CharSeq;
-  typedef sequence<wchar> WcharSeq;
+  typedef sequence<wchar> WCharSeq;
   typedef sequence<boolean> BooleanSeq;
   typedef sequence<octet> ByteSeq;
   // Already defined in DdsDcpsCore.idl
   // typedef sequence<string> StringSeq;
-  typedef sequence<wstring> WstringSeq;
+  typedef sequence<wstring> WStringSeq;
```

Comment:
Is there a spec issue for this?

Reply:
I don't know. I thought it was a typo when it was copied from the spec because the naming convention seems to be `WStringSeq` and `WCharSeq` from other places like CORBA. If it's not a typo, I'll revert these changes.

Reply:
I think we should have this match XTypes Annex C. It's unfortunate that the IDL in that section is inconsistent with other similar IDL in related specs.

Reply:
I've reverted these changes.